google-cloud-spanner/clirr-ignored-differences.xml

```diff
@@ -453,4 +453,71 @@
     <className>com/google/cloud/spanner/TransactionContext</className>
     <method>com.google.api.core.ApiFuture executeUpdateAsync(com.google.cloud.spanner.Statement)</method>
   </difference>
+
+  <!-- Support creating encrypted databases -->
+  <difference>
+    <differenceType>7004</differenceType>
```

Comment:
This change will break customers and requires a major version bump. This is not optional.

Reply:
Modified the clirr differences to include only the interface / implementation changes required for this feature. Since we need to set the encryption info we will need to break the interface here (meaning we will need a major version bump).

Reply:
Marked the PR as breaking.